app/extensions/api.py

```diff
@@ -0,0 +1,167 @@
+from copy import deepcopy
+from functools import wraps
+from flask.globals import current_app, request
+
+import flask_smorest
+import http
+
+
+class ErrorHandlerMixin(flask_smorest.ErrorHandlerMixin):
+    def handle_http_exception(self, error):
+        if error:
+            if hasattr(error, 'data'):
+                error.data['message'] = error.description
+            else:
+                setattr(error, 'data', dict(message=error.description))
+        return super().handle_http_exception(error)
+
+
+class ResponseReferencesPlugin(flask_smorest.spec.plugins.ResponseReferencesPlugin):
+    def _available_responses(self):
+        specs = super()._available_responses()
+        del specs['NOT_MODIFIED']['schema']
+        return specs
+
+
+class Blueprint(flask_smorest.Blueprint):
+    def register_blueprint(self, blueprint, **options) -> None:
+        url_prefix = self.url_prefix + blueprint.url_prefix + options.get('url_prefix', '')
+        return super().register_blueprint(blueprint, **options, url_prefix=url_prefix)
```

Comment:
how does this here work with the "suggested changes"? It's marked as suggestion and can be added/committed as in "I accept the change" with just a click on the button? how did you create the suggestion?

Reply:
First you have to mark the exact lines you want add suggestions to. It's important to click at the `+` at the last selected line. Afterwards you may add on the suggestion icon:
![image](https://user-images.githubusercontent.com/21277928/121529372-a0b39f80-c9fc-11eb-9a2e-7b9ffb9aad0a.png)
This inserts a code section with the selected content. GitHub generates the `Suggested Change` snippet from this section. The comment above is generated from this snippet (with triple backticks instead of ')
```
'''suggestion
    def register_blueprint(self, blueprint, **options):
        url_prefix = self.url_prefix + blueprint.url_prefix + options.get('url_prefix', '')
        super().register_blueprint(blueprint, **options, url_prefix=url_prefix)
'''
```